Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2021"
 [dependencies]
 tokio = { version = "1.24.2", features = ["time"] }
 tracing = "0.1.32"
+rand = "0.8.5"
```

Comment:
I would use [`fastrand`](https://docs.rs/fastrand/latest/fastrand/) if you don't need any advanced features! :D